### PR TITLE
feat(replay): Fetch errors from the issuePlatform as well, so we get Rage and Perf issues on Replay Details

### PR DESCRIPTION
**Pull request by ryan953**
This loads up issuePlatform errors (Rage Clicks and Perf Issues) into the Errors tab on Replay Details. 

It's not linking those Rage Clicks up with existing breadcrumbs, so you'll see similar info in the Breadcrumbs tab. What we could do is followup with some linking, so you can navigate from the breadcrumbs tab into issue details... it's possible now.

| Note | img |
| --- | --- |
| Rage Clicks in Errors tab | SCR-20240312-mfie
| Existing Rage Click in Breadcrumb (unchanged) | SCR-20240312-mfis
| Perf issues in Errors tab | SCR-20240312-mfyn

Depends on https://github.com/getsentry/sentry/pull/66735/files
Relates to https://github.com/getsentry/team-replay/issues/395

### CODE REVIEW

File: static/app/utils/replays/hooks/useReplayData.tsx [hunk 1]
```diff
@@ -140,6 +141,34 @@ function useReplayData({
...
+      finishedAtClone.setSeconds(finishedAtClone.getSeconds() + 1);
+
+      return [
+        `/organizations/${orgSlug}/replays-events-meta/`,
+        {
+          query: {
+            dataset: DiscoverDatasets.ISSUE_PLATFORM,
             start: replayRecord?.started_at.toISOString(),
             end: finishedAtClone.toISOString(),
             project: ALL_ACCESS_PROJECTS,
```
Comment (michellewzhang) [inline static/app/utils/replays/hooks/useReplayData.tsx:174]: noticed that the other `QueryKey` has these extra params as well -- do we need them here too?

```
query: `replayId:[${replayRecord?.id}]`,
            per_page,
            cursor,`
```

Comment (ryan953) [reply static/app/utils/replays/hooks/useReplayData.tsx:174]: yeah, should be.. hrm

**Review by michellewzhang — APPROVED** [review]

File: static/app/utils/replays/hooks/useReplayData.tsx (hunk 1 above)
Comment (ryan953) [reply static/app/utils/replays/hooks/useReplayData.tsx:174]: it's all good, those props do exist in both the places where we call `useFetchSequentialPages`: inside `getErrorsQueryKey` and inside `getPlatformErrorsQueryKey`.

I think the diff algorithm is picking hard to read chunks, but expanding all the lines in the file, and also using Unified diff mode makes it better. 
SCR-20240312-mfyn

Can also look at the file on the branch directly: 
SCR-20240312-nncq

https://github.com/getsentry/sentry/blob/8af2f5ce1a742854ed0530776f9e3753a55da23c/static/app/utils/replays/hooks/useReplayData.tsx

Comment (michellewzhang) [reply static/app/utils/replays/hooks/useReplayData.tsx:174]: oh nice! didn't realize, that's my bad

Comment (ryan953) [reply static/app/utils/replays/hooks/useReplayData.tsx:174]: nbd! i had to look twice too

**Pull request merged**